### ISSUE TITLE
serve_dir script is missing

### ISSUE THREAD

**Issue by eserte** [memowe/Mojo-ServeDir#6]
It seems that a script is missing in the tarball:
```
Output from '/usr/bin/make':

cp lib/ojo/ServeDir.pm blib/lib/ojo/ServeDir.pm
make: don't know how to make script/serve_dir. Stop

make: stopped in /usr/home/cpansand/.cpan/build/2020022203/ojo-ServeDir-0.1-0
```

**Comment by memowe** [comment]
Aah, f*ck, I think I forgot to update the manifest. Will correct it...

Thanks for reporting!

**Comment by memowe** [comment]
@eserte Would you mind trying it again once it's released on CPAN (version 0.11)? 😊

**Comment by eserte** [comment]
Looks good: http://fast-matrix.cpantesters.org/?dist=ojo-ServeDir%200.11
